### ISSUE TITLE
Questions about using this with Spektrum

### ISSUE THREAD

**Issue by RyanEwen** [sebseb7/SbusToPPM#2]
Hello!

I can't find any method of contacting you other than through the issue tracker..

I have used your schematic and code to build my own SBus to Spektrum converter. It's nearly done (board built, chip programmed) and today I did a quick test to see if it would work with my flight controller (which only works with Spektrum receivers). My flight controller is from a Blade QX3 and the stock receiver for it looks the same as a Spektrum remote receiver. 

I want to use my FrSky X8R instead of the Spektrum receiver. This is why I am trying to convert FrSky SBus (same as Futuba?) to Spektrum.

Tonight I will be sniffing the line to see what is output from this board compared to what is output from the stock Spektrum RX. 

I just wanted to know if you had tested and finalized the Spektrum output? I imagine most people use PPM instead. 

Thanks,
Ryan

**Comment by sebseb7** [comment]
yes I tested spektrum output.

do you have an oszilloscope ?

**Comment by RyanEwen** [comment]
Hi and thanks for the reply!

There is an issue with my board, it oscillates but the signal it not output. I cannot even get an LED to light up with it and my own code. So I put it aside and I ordered one of these:

http://www.banggood.com/ATmega328P-Nano-V3-Controller-Board-Compatible-Arduino-p-940937.html

It's arrived and I need to add the components onto it and program it. Since it is 16mhz I will need to make some changes to the code. I am very new to this kind of programming/project (I am a software engineer) so any tips you may have are much appreciated.

Also, is it possible to buy the board you designed somewhere?

**Comment by sebseb7** [comment]
it's not a commercial project. You can send me your board and I will try to fix it. (and describe you any issues I found)

Did you had any issues running my code on an arduino?

**Comment by RyanEwen** [comment]
I think me and my friend probably made a mistake assembling our first board. As I said we can't get it to even run our own simple code to light up an LED. So that is our fault.

While we are trying to figure out our first board, I ordered a premade board (the one linked above from banggood) to try instead so I will see how that goes. This new board is 16mhz and your design is 20mhz so I will need to learn how to update the code for the different timings.

I have no experience in Arduino and ordered the new board without realizing it's an Arduino clone. I planned to use Atmel Studio again. I don't think I should treat it like an Arduino for this.

**Comment by sebseb7** [comment]
just solder a 20Mhz quartz in and you are fine.

**Comment by RyanEwen** [comment]
I was hoping to keep the 16mhz since it's surface mounted and very small and clean. The 20mhz crystal I have is very very big.

**Comment by sebseb7** [comment]
can you just send me the board you soldered ? I'd like to fix it :-)

**Comment by RyanEwen** [comment]
I will take you up on this offer. Can we chat elsewhere? 

My email is ryan (at) ryan-ewen (dot) com
I also use Google Hangouts for instant messaging.

**Comment by RyanEwen** [comment]
Hi again, 

I haven't heard from you. I am still interested in this project :)

**Comment by RyanEwen** [comment]
I found many suppliers of your board! I guess it got popular. I will buy from one of them :)

http://www.readytoflyquads.com/sbus-to-ppm-or-spektrum-serial

http://www.banggood.com/Converter-from-SBUS-to-PPM-or-Spektrum-Serial-p-1016366.html

http://www.aliexpress.com/item/SBUS-TO-PPM-OR-SPEKTRUM-SERIAL/32421262657.html

**Comment by sebseb7** [comment]
haha nice.

they even build a quite old commit.

this one:
https://github.com/sebseb7/eagle/blob/ab4d1bcaeac4c49a2d584f54036f66ff47513380/sbus-ppm/sbus-ppm.brd


2016-04-24 6:06 GMT+02:00 Ryan Ewen <notifications@github.com>:

**Comment by RyanEwen** [comment]
Has anything important changed?

**Comment by sebseb7** [comment]
no, nothing. but I can't see what kind of crystal they are using.

**Comment by RyanEwen** [comment]
My board arrived but it's extremely hard to read the writing on the crystal. T20C is the top line. Up1R is the bottom line.

**Comment by sebseb7** [comment]
Can someone extract the firmware they ship with it?
